api/router.go

```diff
@@ -0,0 +1,24 @@
+package api
+
+import (
+	"database/sql"
+	"fmt"
+	"net/http"
+
+	"github.com/go-chi/chi"
+)
+
+func Router(db *sql.DB) *chi.Mux {
+	chiRouter := chi.NewRouter()
+	chiRouter.Get("/", HomeHandler)
+	chiRouter.NotFound(NotFoundHandler)
+	return chiRouter
+}
+
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Move this to another file named home/homepage/homehandler.go under the api folder